### PR TITLE
add work-optimized server verbiage

### PR DESCRIPTION
**Pull request by reydelarack**
Adding some work-optimized server verbiage and removed inaccurate info about General Purpose servers, as well as eliminating the whole vCPU section as it is obsolete (that section appears to have been written entirely about first-gen).

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-1aa9b91365/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-be6a24be19/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-9c160e248c/how-to/). :bow:

**Comment by catlook** [comment]
@kmsholcomb Edit please?

**Comment by kmsholcomb** [comment]
Sure, @catlook! I'll start on it now.

**Comment by kmsholcomb** [comment]
Hi @catlook. I took the liberty of editing the whole article because it looked like it hadn't been edited in a while (if ever). So, more changes than you probably want! See the attached Word doc (or PDF copy of it) for my suggestions. One overall suggestion that I would make here is that I think the categorization of some of the Q&A pairs could be revisited. I made comments about some specific ones, but I would recommend reviewing all of them to ensure that they are in the proper categories, and that they are in the proper order within each category. You might not have time to do this, and I understand, but I think it would improve the article. Please let me know if you have any questions. Thanks!

[Cloud Servers FAQ_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1008832/Cloud.Servers.FAQ_editsKH.pdf)
[Cloud Servers FAQ_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1008831/Cloud.Servers.FAQ_editsKH.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3a6ef2e545/how-to/)

File: content/cloud-servers/cloud-servers-faq.md [hunk 1]
```diff
@@ -6,106 +6,100 @@ type: article
...
 Administrator access, and are built with nothing but the default
-applications on install. Cloud Servers are geared towards customers who
+applications on installation. Cloud servers are designed for customers who
 want to start from scratch and tune a system for their own purposes.
 
-If, after wearing the Systems Administrator hat for a while, you feel
-that you are out of your league - don't despair! We also offer Cloud
-Servers with a Managed Operations. Read here for more information on
-[Cloud Servers with Managed Operations](http://www.rackspace.com/cloud/managed_cloud/).
+If you are uncomfortable in teh System Administrator role, we also offer a
```
Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: Oh dear! A "teh"!

File: content/cloud-servers/cloud-servers-faq.md [hunk 2]
```diff
@@ -176,423 +182,338 @@ need to configure an A record in Cloud DNS. To access Cloud DNS, in the
...
 communicate between servers. The traffic that flows over this interface
 (eth1) on your server is unmetered and is not billed. This network is
-also referred to as ServiceNet. ServiceNet is an internal only,
-multi-tenant network connection within each Rackspace datacenter.
-ServiceNet IPs are not accessible from the public Internet and are local
-per datacenter.
+referred to as ServiceNet. ServiceNet is an internal-only,
+multitenant network connection within each Rackspace data center.
+ServiceNet IP addresses are not accessible from the public Internet and are local
+per data center. You can also deploy with Cloud Networks, which are single-tenant.
```
Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: In this paragraph we have "multitenant" and "single-tenant". Is there meant to be a hyphen or no? We should probably be consistent.

File: content/cloud-servers/cloud-servers-faq.md [hunk 3]
```diff
@@ -176,423 +182,338 @@ need to configure an A record in Cloud DNS. To access Cloud DNS, in the
...
 will be that your new server has a different IP address from the old
-one. Putting the new server into production may require you to update
+one. Putting the new server into production might require you to update
 any associated DNS records to reflect the new IP address.
 
-#### How do I power my server back on after shutting it down?
+#### How do I power on my server after shutting it down?
 
-Please see the following article: [Shutting Down and Restarting a Cloud Server](/how-to/reboot-your-server "Shutting Down and Restarting a Cloud Server")
+Intructions are  located in [Reboot your server](/how-to/reboot-your-server)
```
Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: There's an extra space between "are" and "located".

File: content/cloud-servers/cloud-servers-faq.md [hunk 4]
```diff
@@ -176,423 +182,338 @@ need to configure an A record in Cloud DNS. To access Cloud DNS, in the
...
 
 #### Which distributions do you offer?
 
-The Features section of our [product page](http://www.rackspace.com/cloud/servers/) has information regarding
-the Linux distributions and Windows operating systems that we have
-available.
+To see the Linux distributions and Windows operating systems that we have
+available, go to the Cloud Control Panel, **Servers>CloudServers**, and
```
Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: Missing some spaces in **Servers > Cloud Servers**

File: content/cloud-servers/cloud-servers-faq.md [hunk 5]
```diff
@@ -176,423 +182,338 @@ need to configure an A record in Cloud DNS. To access Cloud DNS, in the
...
-#### What happens when a Security Group rule is added to the Security Group?
-
-Traffic matching a Security Group rule is allowed to go through.
-
-#### Can I apply Security Groups to ports on an instance at boot time?
+#### Are cloud servers PCI-DSS compliant?
 
-No. Security Groups can only be applied after the instance is active.
+For information about PCI-DSS, see [
+Cloud Security Solutions](https://support.rackspace.com/how-to/are-cloud-servers-pci-dss-compliant/).
```
Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: This link should be simplified to `[Cloud Security Solutions](/how-to/are-cloud-servers-pci-dss-compliant)`

File: content/cloud-servers/cloud-servers-faq.md (hunk 1 above)
Comment (catlook) [inline content/cloud-servers/cloud-servers-faq.md]: doh!

File: content/cloud-servers/cloud-servers-faq.md (hunk 2 above)
Comment (catlook) [inline content/cloud-servers/cloud-servers-faq.md]: Kelly pulled the hyphen from multitenant.

Comment (stephamon) [inline content/cloud-servers/cloud-servers-faq.md]: Fair enough!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3622533c2c/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**